cmd/orpheus/queue.go

```diff
@@ -6,21 +6,31 @@ import (
 	"math/rand"
 )
 
-func (server *Server) Add(url string, userId string) (*QueueItem, error) {
-	song, err := fetchSongFromURL(url)
+func (server *Server) Add(url string, userId string, shuf bool) ([]*QueueItem, error) {
```

Comment:
This is personal preference, but I think the `shuffle` instead of `shuf` is nicer.